_config.yml

```diff
@@ -6845,45 +6941,45 @@ ontologies:
     label: Zea
   title: Maize gross anatomy
 principles:
-- id: fp-000-summary
-  layout: principle
-  title: Overview
-- id: fp-001-open
+- id: fp-004-versioning
```

Comment:
Somehow the order of the principles got randomised, is this a problem?

Reply:
That's annoying. I think I fixed it.